src/common/configs/needs-review-rule-resources.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const needsReviewRuleResourcesPath =
+    'https://accessibilityinsights.io/info-examples/web/needs-review';
+
+export const isOutcomeNeedsReview = (outcomeType: string) => {
```

Comment:
Can we add the output type here?
```suggestion
export const isOutcomeNeedsReview = (outcomeType: string) : boolean => {
```

Reply:
resloved